cylance_protect/icon_cylance_protect/actions/blacklist/action.py

```diff
@@ -0,0 +1,39 @@
+import insightconnect_plugin_runtime
+from .schema import BlacklistInput, BlacklistOutput, Input, Output, Component
+from insightconnect_plugin_runtime.exceptions import PluginException
+# Custom imports below
+import re
+
+
+class Blacklist(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='blacklist',
+                description=Component.DESCRIPTION,
+                input=BlacklistInput(),
+                output=BlacklistOutput())
+
+    def run(self, params={}):
+        sha1_hash = params.get(Input.HASH)
```

Comment:
@r7-kszczepanskagorna Can add code to check if the hash is SHA256? and print a friendly error message that SHA256 is only supported if the user passes a different input.